test-suite/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/LocalDateTest.scala

```diff
@@ -0,0 +1,790 @@
+      } {
+        expectThrows[UnsupportedTemporalTypeException](d1.until(d2, u))
+      }
+
+      expect(MIN.until(MAX) == Period.of(1999999998, 11, 30)).toBeTruthy
+      expect(someDate.until(leapDate) == Period.of(1, 0, 1)).toBeTruthy
+      expect(leapDate.until(of(2013, 2, 28)) == Period.of(0, 11, 30)).toBeTruthy
+      expect(leapDate.until(of(2013, 3, 1)) == Period.of(1, 0, 1)).toBeTruthy
+
+      for {
+        d1 <- samples1
+        d2 <- samples1 if d2.isAfter(d1)
+      } {
+        expect(d2.until(d1) == d1.until(d2).negated).toBeTruthy
+      }
+    }
+
+    it("should respond to `toEpochDay`") {
+      expect(MIN.toEpochDay == -365243219162L).toBeTruthy
+      expect(of(1969, 12, 31).toEpochDay == -1).toBeTruthy
+      expect(of(1970, 1, 1).toEpochDay == 0).toBeTruthy
+      expect(someDate.toEpochDay == 15033).toBeTruthy
+      expect(leapDate.toEpochDay == 15399).toBeTruthy
+      expect(MAX.toEpochDay == 365241780471L).toBeTruthy
+    }
+
+    it("should be comparable") {
+      expect(MIN.compareTo(MIN)).toEqual(0)
+      expect(MIN.compareTo(someDate)).toBeLessThan(0)
+      expect(MIN.compareTo(MAX)).toBeLessThan(0)
+      expect(someDate.compareTo(MIN)).toBeGreaterThan(0)
+      expect(someDate.compareTo(someDate)).toEqual(0)
+      expect(someDate.compareTo(MAX)).toBeLessThan(0)
+      expect(MAX.compareTo(MIN)).toBeGreaterThan(0)
+      expect(MAX.compareTo(someDate)).toBeGreaterThan(0)
+      expect(MAX.compareTo(MAX)).toEqual(0)
+    }
+
+    it("should respond to `isAfter`") {
+      expect(MIN.isAfter(MIN)).toBeFalsy
+      expect(MIN.isAfter(someDate)).toBeFalsy
+      expect(MIN.isAfter(MAX)).toBeFalsy
+      expect(someDate.isAfter(MIN)).toBeTruthy
+      expect(someDate.isAfter(someDate)).toBeFalsy
+      expect(someDate.isAfter(MAX)).toBeFalsy
+      expect(MAX.isAfter(MIN)).toBeTruthy
+      expect(MAX.isAfter(someDate)).toBeTruthy
+      expect(MAX.isAfter(MAX)).toBeFalsy
+    }
+
+    it("should respond to `isBefore`") {
+      expect(MIN.isBefore(MIN)).toBeFalsy
+      expect(MIN.isBefore(someDate)).toBeTruthy
+      expect(MIN.isBefore(MAX)).toBeTruthy
+      expect(someDate.isBefore(MIN)).toBeFalsy
+      expect(someDate.isBefore(someDate)).toBeFalsy
+      expect(someDate.isBefore(MAX)).toBeTruthy
+      expect(MAX.isBefore(MIN)).toBeFalsy
+      expect(MAX.isBefore(someDate)).toBeFalsy
+      expect(MAX.isBefore(MAX)).toBeFalsy
+    }
+
+    it("should override `toString`") {
+      expect(MIN.toString).toEqual("-999999999-01-01")
+      expect(of(-1, 12, 31).toString).toEqual("-0001-12-31")
+      expect(of(0, 1, 1).toString).toEqual("0000-01-01")
+      expect(someDate.toString).toEqual("2011-02-28")
+      expect(leapDate.toString).toEqual("2012-02-29")
+      expect(of(9999, 12, 31).toString).toEqual("9999-12-31")
+      expect(of(10000, 1, 1).toString).toEqual("+10000-01-01")
+      expect(MAX.toString).toEqual("+999999999-12-31")
+    }
+
+    it("should respond to `now`") {
+      now()
```

Comment:
There is not much that can be tested with `now()`, but we know that it should at least be non-null.
You may also try to test the era, I'm assuming that every machine should tell that the era is `CE`.